frontend/public/locales/en/translation.json

```diff
@@ -2574,7 +2574,7 @@
     "To apply a resource to a cluster, the placement must be bound to the resource using a placement binding.": "To apply a resource to a cluster, the placement must be bound to the resource using a placement binding.",
     "To complete these values, use the imageContentSources that you recorded during mirror registry creation.": "To complete these values, use the imageContentSources that you recorded during mirror registry creation.",
     "To create an infrastructure environment, you must configure the host inventory settings.": "To create an infrastructure environment, you must configure the host inventory settings.",
-    "to create the hosted control plane command.": "to create the hosted control plane command.",
+    "to create the Hosted Control Plane command..": "to create the Hosted Control Plane command.",
```

Comment:
Pushed a commit to fix this typo (extra period in key)

Always use the tooling to prevent errors like this.